packages/url/src/test/index.test.js

```diff
@@ -69,6 +72,60 @@ describe( 'addQueryArgs', () => {
 	} );
 } );
 
+describe( 'getQueryArg', () => {
+	it( 'should get the value of an existing query arg', () => {
+		const url = 'https://andalouses.example/beach?foo=bar&bar=baz';
+
+		expect( getQueryArg( url, 'foo' ) ).toBe( 'bar' );
+	} );
+
+	it( 'should not return a value of an unknown query arg', () => {
+		const url = 'https://andalouses.example/beach?foo=bar&bar=baz';
+
+		expect( getQueryArg( url, 'baz' ) ).toBeUndefined();
+	} );
+
+	it( 'should get the value of an arry query arg', () => {
+		const url = 'https://andalouses.example/beach?foo[]=bar&foo[]=baz';
+
+		expect( getQueryArg( url, 'foo' ) ).toEqual( [ 'bar', 'baz' ] );
+	} );
+} );
+
+describe( 'hasQueryArg', () => {
+	it( 'should return true for an existing query arg', () => {
+		const url = 'https://andalouses.example/beach?foo=bar&bar=baz';
+
+		expect( hasQueryArg( url, 'foo' ) ).toBeTruthy();
+	} );
+
+	it( 'should return false for an unknown query arg', () => {
+		const url = 'https://andalouses.example/beach?foo=bar&bar=baz';
+
+		expect( hasQueryArg( url, 'baz' ) ).toBeFalsy();
+	} );
+
+	it( 'should return true for an arry query arg', () => {
+		const url = 'https://andalouses.example/beach?foo[]=bar&foo[]=baz';
+
+		expect( hasQueryArg( url, 'foo' ) ).toBeTruthy();
+	} );
+} );
+
+describe( 'removeQueryArgs', () => {
```

Comment:
Can we add a test covering array parameters.

Reply:
Added!